spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -44,18 +44,38 @@ describe 'orifice-calculator-viewmodel-spec', ->
       expect(viewModel.pipeID().length).toEqual 6
 
     it 'should have default value of "1.939\'\' XS, Sch 80, Sch 80S"', ->
-      expect(viewModel.selectedPipeID()).toEqual '1.939\'\' XS, Sch 80, Sch 80S'
+      expect(viewModel.selectedPipeID()).toEqual OPL.OrificeCalculator.Config.Dictionaries.PipeID.oneNineInch
 
   describe 'operatingPressure', ->
     itBehavesLikeMandatoryField viewModel.operatingPressure
     itBehavesLikeIntegerField viewModel.operatingPressure
 
+    describe 'when pressure is high', ->
+
+      it 'should display a operatingPressureWarning message at > 200', ->
+        viewModel.operatingPressure(201)
+        viewModel.operatingPressure.valueHasMutated()
+        expect(viewModel.operatingPressureWarningMessage()).toEqual OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningMayResult
+
+      it 'should display a operatingPressureWarning message at < 401', ->
+        viewModel.operatingPressure(400)
+        expect(viewModel.operatingPressureWarningMessage()).toEqual OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningMayResult
+
+      it 'should display an operatingPressureError message at > 400', ->
+        viewModel.operatingPressure(401)
+        expect(viewModel.operatingPressureWarningMessage()).toEqual OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarningWillResult
+
+    describe 'when pressure is low', ->
+      it 'should not have any messages', ->
+        viewModel.operatingPressure(100)
+        expect(viewModel.operatingPressure.isValid()).toEqual true
```

Comment:
This is no longer being used. Check that the operatingPressureWarningMessage is not defined